spec/feature/hearings/virtual_hearings/hearing_details_spec.rb

```diff
@@ -69,6 +69,51 @@ def check_email_event_table(hearing, row_count)
   let(:pre_loaded_rep_email) { hearing.appeal.representative_email_address }
   let(:fill_in_veteran_email) { "email@testingEmail.com" }
 
+  def check_virtual_hearings_links_expired(virtual_hearing)
+    within "#vlj-hearings-link" do
+      expect(page).to have_content(
+        "VLJ Link: Expired\n" \
+        "Conference Room: BVA#{virtual_hearing.alias}@care.va.gov\n" \
+        "PIN: #{virtual_hearing.host_pin}"
+      )
+    end
+    within "#guest-hearings-link" do
+      expect(page).to have_content(
+        "Guest Link: Expired\n" \
+        "Conference Room: BVA#{virtual_hearing.alias}@care.va.gov\n" \
+        "PIN: #{virtual_hearing.guest_pin}"
+      )
+    end
+  end
+
+  def check_virtual_hearings_links(virtual_hearing, label)
+    # Construct the Virtual Hearing Links
+    client_host = ENV["PEXIP_CLIENT_HOST"] || "care.evn.va.gov"
+    base_link = "https://#{client_host}/bva-app/?conference=#{virtual_hearing.alias}&"
```

Comment:
You don't need this since there's a` pexip_url` which stubs the `client_host` env in line 60. Also you can change the use `formatted_alias_or_alias_with_host` here since the virtual hearing does not have `alias_with_host`

```suggestion
    base_link = "https://#{pexip_url}/bva-app/?conference=#{virtual_hearing.formatted_alias_or_alias_with_host}&"
```